front/src/app/groups/[id]/page.tsx

```diff
@@ -45,64 +329,504 @@ const GroupViewPage = () => {
+                          margin="auto"
+                          marginLeft="0"
+                          paddingTop="0px"
+                          marginTop="0px"
+                          position="sticky"
+                          sx={{
+                            [theme.breakpoints.down("md")]: {
+                              width: "100%",
+                            },
+                          }}
+                        >
+                          <TextField
+                            InputProps={{
+                              startAdornment: (
+                                <SearchIcon
+                                  sx={{
+                                    color: "grey",
+                                  }}
+                                />
+                              ),
+                            }}
+                            placeholder="Search users"
+                            variant="outlined"
+                            value={userSearchTerm}
+                            onChange={handleUserSearchChange}
+                            size="small"
+                            sx={{
+                              width: {
+                                xs: "100%",
+                                sm: "75%",
+                                md: "50%",
+                                lg: "340px",
+                              },
+                              height: "40px",
+                              "& .MuiInputBase-root": {
+                                height: "100%",
+                                outline: "none",
+                                textDecoration: "none",
+                              },
+                            }}
+                          />
+                        </Box>
+                      </TableCell>
+                    </TableRow>
+                  </TableHead>
+                  <TableBody>
+                    {loading && (
+                      <div style={{ textAlign: "center", marginTop: "5%" }}>
+                        <CircularProgress />
+                      </div>
+                    )}
+                    {!loading && (
+                      <TableRow sx={{ marginTop: "0px" }}>
+                        <TableCell colSpan={2}>
+                          <TableContainer
+                            component={Paper}
+                            sx={{
+                              width: "100%",
+                              height: "260px",
+                              marginTop: "2px",
+                              marginBottom: "0px",
+                              overflowY: "auto",
+                              border: "none",
+                              boxShadow: "none",
+                            }}
+                          >
+                            <FormGroup>
+                              {userOptions
+                                .filter((option) =>
+                                  option.firstName
```

Comment:
Handle "TypeError: Cannot read properties of null" on runtime

Reply:
TypeError has been solved